src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```
@@ -1,16 +1,27 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(SessionFactory sessionFactory) {
+        this.actorDao = new ActorDaoImpl(sessionFactory);
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+
+        return actorDao.get(id).orElse(null);
```

Consider handling the case where the `Optional` is empty more explicitly, possibly by throwing an exception or returning a default value, instead of returning `null`. This can help avoid potential `NullPointerExceptions` in the calling code.